org.thingpedia.iot.garage-door/dataset.tt

```diff
@@ -0,0 +1,78 @@
+dataset @org.thingpedia.iot.garage-door language "en" {
```

Comment:
Didn't we discuss merging door with garage door?

Reply:
Merged in latest commit